Use getrandom to retrieve random bytes

ueno opened the pull request:
This is to avoid extra DRBG logic to be compiled at runtime.  When compiled against the `wasm32-wasi` target, the Rust `getrandom` call is delegated to `getrandom(..., GRND_NONBLOCK)`, which implements the same logic in the [Linux kernel](https://www.chronox.de/lrng/doc/lrng.html).

npmccallum commented:
After discussion with the team, we have decided to keep the default `rustfmt` style. We concluded that there was value in not deviating from what the broader Rust community is doing.

We are, however, adding a test to ensure that the style is correct on each PR. This should help us maintain consistency.

Therefore, we are rejecting this PR.

ueno committed:
Use getrandom to retrieve random bytes

This is to avoid extra DRBG logic to be compiled at runtime.  When
compiled against wasm32-wasi target, the Rust `getrandom` call is
delegated to getrandom(..., GRND_NONBLOCK), which implements the same
logic in the Linux kernel:
https://www.chronox.de/lrng/doc/lrng.html
